src/main/java/uk/gov/beis/els/api/categories/waterheaters/ConventionalWaterHeaterApiFormSequenceProvider.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.beis.els.api.categories.waterheaters;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.hibernate.validator.spi.group.DefaultGroupSequenceProvider;
+import uk.gov.beis.els.categories.waterheaters.model.EnergyConsumptionUnit;
+import uk.gov.beis.els.categories.waterheaters.model.validation.ConsumptionUnitBoth;
+import uk.gov.beis.els.categories.waterheaters.model.validation.ConsumptionUnitGj;
+import uk.gov.beis.els.categories.waterheaters.model.validation.ConsumptionUnitKw;
+
+public class ConventionalWaterHeaterApiFormSequenceProvider implements DefaultGroupSequenceProvider<ConventionalWaterHeaterApiForm> {
```

Comment:
Do we need new sequence providers here? They look to be identical to the existing ones, e.g `ConventionalWaterHeaterFormSequenceProvider` has the same logic